### PR TITLE
Let the voice message duration grow to fit a wider font

### LINKED ISSUE #18590: Voice message duration indicator doesn't fit in UI when using a custom font
### Steps to reproduce

1. Configure a custom font such as `monospace` in element desktop (without selecting a different font size etc..).
2. Send or receive a voice message
3. Look at the rendered voice message item

### What happened?

image

The duration indicator did flow over into the next line. That seems to be due to the width of the item being relative to the text size on the `<html>` node and not relative to the size of the actual font used(?).

### What did you expect?

The size of the field should be relative to the size of the font in the field (up to a reasonable limit, e.g. the full width).

### Operating system

MacOS

### Application version

Element desktop 1.8.1

### How did you install the app?

https://packages.riot.im/desktop/install/macos/Element.dmg

### PR DESCRIPTION
**Pull request by hayaksi1**
The voice message duration is given a width derived from the root font size, with a comment saying
it fakes a monospace advance. A custom font with wider digits does not fit that width, and since
the player is painted with `contain: content` the duration ends up wrapping or clipped.

The fixed value now acts as a minimum rather than a fixed size, so the box keeps its usual width
with the default font and grows only when the digits need more room. The floor that keeps the
layout steady in cramped flexboxes, such as the file panel, is retained.

Tests: a Playwright case that widens the clock's digits and asserts the duration still renders on
one line inside its box. The existing voice message screenshot is unchanged.

Fixes #18590

## Checklist

- [x] I have read through [review guidelines](https://github.com/element-hq/element-web/blob/develop/docs/review.md) and [CONTRIBUTING.md](https://github.com/element-hq/element-web/blob/develop/CONTRIBUTING.md).
- [x] Tests written for new code (and old code if feasible).
- [x] New or updated `public`/`exported` symbols have accurate [TSDoc](https://tsdoc.org/) documentation.
- [x] Linter and other CI checks pass.
- [x] I have licensed the changes to Element by completing the [Contributor License Agreement (CLA)](https://cla-assistant.io/element-hq/element-web)

### CODE REVIEW

**Comment by dbkr** [comment]
Thanks for adding the test, although in this case if the only way to test this is by adding a playwright test that has to wait for 3 seconds, I think I'd make an exception for this edge case and say it's probably fine to go in without a test. Our playwright tests take a long time so we do need to be a bit careful about what we add playwright tests for.

File: apps/web/playwright/e2e/timeline/timeline.spec.ts [hunk 1]
```diff
@@ -1040,6 +1040,58 @@ test.describe("Timeline", () => {
...
         });
 
+        test("shows the whole voice message duration with a wide font", async ({ page, app, room, context }) => {
+            await app.viewRoomById(room.roomId);
+
+            const composerOptions = await app.openMessageComposerOptions();
+            await composerOptions.getByRole("menuitem", { name: "Voice Message" }).click();
+
+            // Record an empty message
+            await page.waitForTimeout(3000);
```
Comment (Half-Shot) [inline apps/web/playwright/e2e/timeline/timeline.spec.ts]: As per @dbkr 's comment about timing, can we instead fake this by sending a matrix event that looks like a voicenote into the room? 

The asserts below seem to imply we aren't testing the recording parts at all.

**Review by Half-Shot — CHANGES_REQUESTED** [review]
Tests need to change, then this can go in.

File: apps/web/playwright/e2e/timeline/timeline.spec.ts (hunk 1 above)
Comment (hayaksi1) [reply apps/web/playwright/e2e/timeline/timeline.spec.ts]: Done — it now uploads `1sec.ogg` and sends an `m.audio` event carrying the msc3245 voice marker, so there is no recorder and no 3s wait; it runs in about 4s. Moved under "message displaying" since it only measures layout. Verified still red against the old fixed width.

**Review by Half-Shot — APPROVED** [review]
